checker/src/main/java/org/checkerframework/checker/nullness/KeyForPropagator.java

```diff
@@ -178,23 +178,30 @@ public void propagateNewClassTree(
      * An annotated type merger that merges @KeyFor annotations and only if the type that is
      * receiving an annotation has an @UnknownKeyFor annotation or NO key for annotations.
      */
-    private class KeyForPropagationMerger extends AnnotatedTypeMerger {
+    private class KeyForPropagationReplacer extends AnnotatedTypeReplacer {
+
+        /** The processing environment. */
         private final ProcessingEnvironment env;
 
-        private KeyForPropagationMerger(ProcessingEnvironment env) {
+        /**
+         * Create a new replacer.
+         *
+         * @param env the processing environment
+         */
+        private KeyForPropagationReplacer(ProcessingEnvironment env) {
             this.env = env;
         }
 
         @Override
-        protected void replaceAnnotations(AnnotatedTypeMirror from, AnnotatedTypeMirror to) {
-            final AnnotationMirror fromKeyFor = from.getAnnotationInHierarchy(UNKNOWN_KEYFOR);
-            final AnnotationMirror toKeyFor = to.getAnnotationInHierarchy(UNKNOWN_KEYFOR);
+        protected void replaceAnnotations(AnnotatedTypeMirror src, AnnotatedTypeMirror dst) {
```

Comment:
The PR would be nicer if these parameter renamings happened separately, to focus on the essential changes.